brooklyn-server/server-cli/src/main/java/org/apache/brooklyn/cli/ItemLister.java

```diff
@@ -98,88 +115,150 @@ public Void call() throws Exception {
-                copyFromItemListerClasspathBaseStaticsToOutputDir(resourceUtils, "style/js/underscore-min.map");
-                copyFromItemListerClasspathBaseStaticsToOutputDir(resourceUtils, "style/js/catalog/typeahead.js");
-
-                // now make pages for each item
-                
-                List<Map<String, Object>> entities = (List<Map<String, Object>>) result.get("entities");
-                String entityTemplateHtml = resourceUtils.getResourceAsString(Urls.mergePaths(BASE_TEMPLATES, "entity.html"));
-                for (Map<String, Object> entity : entities) {
-                    String type = (String) entity.get("type");
-                    String name = (String) entity.get("name");
-                    String entityHtml = TemplateProcessor.processTemplateContents(entityTemplateHtml, ImmutableMap.of("type", type, "name", name));
-                    Files.write(entityHtml, new File(Os.mergePaths(outputFolder, "entities", type + ".html")), Charsets.UTF_8);
-                }
-                
-                List<Map<String, Object>> policies = (List<Map<String, Object>>) result.get("policies");
-                String policyTemplateHtml = resourceUtils.getResourceAsString(Urls.mergePaths(BASE_TEMPLATES, "policy.html"));
-                for (Map<String, Object> policy : policies) {
-                    String type = (String) policy.get("type");
-                    String name = (String) policy.get("name");
-                    String policyHtml = TemplateProcessor.processTemplateContents(policyTemplateHtml, ImmutableMap.of("type", type, "name", name));
-                    Files.write(policyHtml, new File(Os.mergePaths(outputFolder, "policies", type + ".html")), Charsets.UTF_8);
-                }
-                
-                List<Map<String, Object>> enrichers = (List<Map<String, Object>>) result.get("enrichers");
-                String enricherTemplateHtml = resourceUtils.getResourceAsString(Urls.mergePaths(BASE_TEMPLATES, "enricher.html"));
-                for (Map<String, Object> enricher : enrichers) {
-                    String type = (String) enricher.get("type");
-                    String name = (String) enricher.get("name");
-                    String enricherHtml = TemplateProcessor.processTemplateContents(enricherTemplateHtml, ImmutableMap.of("type", type, "name", name));
-                    Files.write(enricherHtml, new File(Os.mergePaths(outputFolder, "enrichers", type + ".html")), Charsets.UTF_8);
+                for (URL url: urls) {
+                    List<Map<?,?>> entities = new ArrayList<>();
+                    List<Map<?,?>> policies = new ArrayList<>();
+                    List<Map<?,?>> enrichers = new ArrayList<>();
+                    List<Map<?,?>> locations = new ArrayList<>();
+                    List<Map<?,?>> locationResolvers = new ArrayList<>();
+
+                    String yamlContent = Files.toString(new File(url.getFile()), Charset.forName("UTF-8"));
```

Comment:
You could treat `url` as a generic URL, so the app can fetch remote catalogs as well. Changing it to something like `Streams.readFullyString(url.openStream)` will be independent of the location.